### PR TITLE
Scaling Section Review

### PR DESCRIPTION
**Pull request by jarolrod**
Various fix-ups and re-wording throughout the scaling section. Changes broken up into separate commits for easy review.

Note to reviewer: please check render of the changes is fine, I did not test for this.

### CODE REVIEW

File: scaling.adoc [hunk 1]
```diff
@@ -235,37 +234,36 @@ different versions of Bitcoin Core going back to version 0.8.
...
+The graph shows the trend for two different 64-bit CPU types, ARM and x86.
+The difference in performance is due to the more specialized instructions
+available on x86 compared to the ARM architecture, which has fewer,
+more generic instructions. But the general trend is the same for both
+architectures. Note that the Y-axis is logarithmic, which makes the
+improvements look less impressive than they actually are.
 
-There are also several good examples of where space savings have
-contributed to performance improvements. In a
+TThere are also several good examples of space savings contributing to
```
Comment (kallerosenbaum) [inline scaling.adoc:265]: TT

File: scaling.adoc [hunk 2]
```diff
@@ -340,23 +338,23 @@ theme is that the participants create a number of semi-signed
...
-We won't go into more detail on how specific protocols work, but as
+We won't detail how specific protocols work, but as
 you can see in <<fig-scaling-layer>>, the blockchain is rarely used
 during the protocol's life cycle. All the juicy action happens
 _off-chain_. We've seen how this can be a win for <<privacy,privacy>>
 if done right, but it can also be a big win for scalability.
 
 In a https://www.reddit.com/r/Bitcoin/comments/438hx0/a_trip_to_the_moon_requires_a_rocket_with/[Reddit post] titled "`A trip to the moon requires a rocket with
-multiple stages or otherwise the rocket equation will eat your
+multiple stages or otherwise, the rocket equation will eat your
```
Comment (kallerosenbaum) [inline scaling.adoc:357]: This is a quote and should not be altered.

**Review by kallerosenbaum — CHANGES_REQUESTED** [review]

**Comment by jarolrod** [comment]
updated from 5bab192 to 0e8d06c ([pr28.01](https://github.com/jarolrod/btcphilosophy/tree/pr28.01) -> [pr28.02](https://github.com/jarolrod/btcphilosophy/tree/pr28.02), [diff](https://github.com/jarolrod/btcphilosophy/compare/pr28.01..pr28.02))

addressed @kallerosenbaum comments ([1](https://github.com/kallerosenbaum/btcphilosophy/pull/28#discussion_r874567014), [2](https://github.com/kallerosenbaum/btcphilosophy/pull/28#discussion_r874572178))

File: scaling.adoc (hunk 2 above)
Comment (jarolrod) [reply scaling.adoc]: thanks, addressed in https://github.com/kallerosenbaum/btcphilosophy/commit/0e8d06c4d600577181c6e1cd933bfc50db412809

File: scaling.adoc (hunk 1 above)
Comment (jarolrod) [reply scaling.adoc]: thanks, addressed in https://github.com/kallerosenbaum/btcphilosophy/commit/0e8d06c4d600577181c6e1cd933bfc50db412809

**Pull request merged**